webui/react/src/e2e/models/README.md

```diff
@@ -58,14 +58,23 @@ All pages and components support subcomponents using instance properties. Inital
 Here's a simplified example using part of the `DeterminedAuth` component. Intrinsic elements are be represent with `BaseComponent`, and they can be as `parent`s for other elements. The amount of specificity is left to the author's discretion. If you're not sure, more is better to avoid conflicts with future additions. Deeper specificity will also optimize for searching through the DOM on larger pages.
 
 ```js
-export class DeterminedAuth extends NamedComponent({ defaultSelector: "div[data-test-component='detAuth']"}) {
+export class DeterminedAuth extends NamedComponent({ defaultselector: `div[data-test-component="detAuth"]`}) {
   constructor({ selector, parent }: NamedComponentArgs) {
     super({ parent: parent, selector: selector || DeterminedAuth.defaultSelector });
   }
-  readonly form: BaseComponent = new BaseComponent({ parent: this, selector: "form" });
-  readonly username: BaseComponent = new BaseComponent({ parent: this.form, selector: "input[data-testid='username']" });
-  readonly password: BaseComponent = new BaseComponent({ parent: this.form, selector: "input[data-testid='password']" });
-  readonly submit: BaseComponent = new BaseComponent({ parent: this.form, selector: "button[data-testid='submit']" });
+  readonly #form: BaseComponent = new BaseComponent({ parent: this, selector: `form` });
```

Comment:
For my own edification, why did this switch to having `#` starting the variable?

Reply:
# is for private.
I was thinking it's useful for the hierarchy, but we never .click() or .fill() it's pwlocator.